tests/test_fft.py

```diff
@@ -181,3 +219,12 @@ def test_vmap():
       atol=1e-10)
   # Check the reverse FFT
   assert_allclose(array, rec_array, rtol=1e-10, atol=1e-10)
+
+# find a way to finalize pytest
+def test_end():
+  # Make sure that it is cleaned up
+  # This has to be this way because pytest runs the "global code" before running the tests
+  # There are other solutions https://stackoverflow.com/questions/41871278/pytest-run-a-function-at-the-end-of-the-tests
```

Comment:
What about defining some fixtures with `scope=session` in a conftest.py ?

This could also be useful for the initialize at the beginning..